Update QCity documentation with development sites information (#64)
north-road/QCity: docs/src/user_guide/tools.md

A project area can be renamed by left clicking on the project area in the Project Setup panel and left clicking on the pencil and paper button on the right hand side of the project areas panel.

![Project_Settings_Rename_Project_Area](https://github.com/user-attachments/assets/4ea00ba2-a6e5-499e-9432-a9999ac81cb0)

## Development Sites

Each development site has the following settings that can be configured:

1. General Details
2. Development Statistics
3. Car Parking Statistics
4. Bike Parking Statistics

### Relation to Project Area

> The development sites tab includes the name of the project area which it is associated with under the development sites heading. This provides you with an indication which project area you are creating the development site within.

Reviewer: ```suggestion
The `Development Sites` tab includes the name of the project area which it is associated with under the development sites heading. This provides you with an indication which project area you are creating the development site within.
```

![Development_Sites_Heading](https://github.com/user-attachments/assets/c0f7d230-d567-4109-a28d-bedaa6c459db)

Should you digitize a development site outside of the project area boundary, QCity will present a warning that the development site is outside of the spatial extent of the project area.

![Development_Sites_Warning_Spatial_Extent](https://github.com/user-attachments/assets/76a03016-7926-4b65-b160-3f1ba4901bf8)

> The development site is linked to the project area that is selected in the projects tab. When working with multiple project areas, always ensure you check the header of the development sites tab to see which project area it is associated with.

### Digitizing Development Sites - Snapping

When creating new development sites, often the boundaries correspond to existing cadastral boundaries. In order to digitize a development site to the boundaries of cadastral lots or other features, snapping needs to be enabled.

> Snapping options are available in the Snapping toolbar.

Reviewer: ```suggestion
Snapping options are available in the QGIS Snapping toolbar.
```

![Enable_Snapping](https://github.com/user-attachments/assets/ecf75a98-b8c5-46ba-8391-d4113eae3c6c)

To enable snapping to a specific layer, left click on the snapping options dropdown icon situated to the right of the `Enable Snapping` button. Left click on the `Advanced Configuration`.

![Snapping_Options_Menu](https://github.com/user-attachments/assets/e338b263-3f79-45c7-8ca7-0677c1e5617e)

Once the Snapping toolbar is set to advanced configuration, left click on the `Edit Advanced Configuration` button which appears as an eye icon. Left click on the checkbox next to the layer you would like to snap to and from the `Type` dropdown, select whether you would like to snap to vertices, segments etc.

![Snapping_Advanced_Configuration](https://github.com/user-attachments/assets/b10886f2-1eab-4a1c-a6d3-26c94ad44430)

Once snapping has been enabled, when created a new development site, you will see a purple indicator on the map canvas to snap to the desired features.

![Snapping_Indicator](https://github.com/user-attachments/assets/0120e5af-4b45-4a2a-b97c-1e9487a403f5)

> > For more information on using snapping within QGIS, please see the QGIS ![documentation](https://docs.qgis.org/3.40/en/docs/user_manual/working_with_vector/editing_geometry_attributes.html#snapping-and-digitizing-options).

Reviewer: ```suggestion
> For more information on using snapping within QGIS, please see the QGIS ![documentation](https://docs.qgis.org/latest/en/docs/user_manual/working_with_vector/editing_geometry_attributes.html#snapping-and-digitizing-options).
```

### General Details

The general details section of the developent sites tab allows you to input the following:

1. Location (Address)
2. Status - Drop down list including:
    1. `Constructed` for modeling of development sites already constructed
    2. `Proposed` for development sites where an approval has been issued but no construction has commenced

>     3. `Modelled` for development sites which are hypothetical developments yet to occur

Reviewer: ```suggestion
    3. `Modeled` for development sites which are hypothetical developments yet to occur
```
